### PR TITLE
Improvement/ansible doc

### PR DESCRIPTION
**Pull request by felddy**
- Add an example of using the Ansible script module to run the script remotely. 
- Pulled in upstream changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,14 +58,23 @@ available without a lengthy install process.  There is an embedded hash
 list this file that can be easily edited.  This makes running the script
 with tools like [Ansible](https://www.ansible.com) much simpler.
 
+Here is an example of running the script remotely using the Ansible
+[script module](https://docs.ansible.com/ansible/latest/modules/script_module.html):
+
+```console
+ansible --inventory=hosts-file cool-servers --module-name=script \
+--args="src/ioc_scan/ioc_scanner.py" --become --ask-become-pass \
+--user="ian.kilmister"
```
Comment (jsf9k) [inline README.md:67]: Nice use of his real name!

**Review by jsf9k — APPROVED** [review]
Powerful improvements

**Review by mcdonnnj — APPROVED** [review]
Illuminating changes.

**Review by dav3r — APPROVED** [review]
Totally gnarly, dude! ⚙️

**Comment by felddy** [comment]
The smarm is strong with this team. 
I'm very proud.

**Pull request merged**